migrating/ossm-migrating-read-me-assembly.adoc

```diff
@@ -0,0 +1,95 @@
+:_content-type: ASSEMBLY
+[id=ossm-migrating-read-me-assembly_{context}]
```

Comment:
Also per the guidelines, the top ID of an assembly is the one exception to IDs needing a context:
```suggestion
[id=ossm-migrating-read-me-assembly]
```